assets/src/blocks/SecondaryNavigation/SecondaryNavigationEditor.js

```diff
@@ -0,0 +1,65 @@
+import {useEffect} from '@wordpress/element';
+import {getHeadingsFromBlocks} from '../TableOfContents/getHeadingsFromBlocks';
+import {initializeJustifyContentAdjustment} from './adjustNavWidth';
+
+const {useSelect} = wp.data;
+// const {InspectorControls} = wp.blockEditor;
+// const {PanelBody} = wp.components;
+const {__} = wp.i18n;
+
+// const renderEdit = () =>  (
+//   // TODO: Update with correct link
+//   // <InspectorControls>
+//   //   <PanelBody title={__('Learn more about this block', 'planet4-blocks-backend')} initialOpen={false}>
+//   //     <p className="components-base-control__help">
+//   //       <a target="_blank" href="https://planet4.greenpeace.org/content/blocks/table-of-contents/" rel="noreferrer">
+//   //           P4 Handbook - P4 Secondary Navigation Menu
+//   //       </a>
+//   //       {' '} &#128203;
+//   //     </p>
+//   //   </PanelBody>
+//   // </InspectorControls>
+// );
+
+const renderView = ({levels, isExample, exampleMenuItems}) => {
+
+  const blocks = useSelect(select => select('core/block-editor').getBlocks(), []);
+
+  useEffect(initializeJustifyContentAdjustment, []);
+
+  const flatHeadings = getHeadingsFromBlocks(blocks, levels);
+
+  const menuItems = isExample ? exampleMenuItems : flatHeadings;
+
+  return (
+    <section className="block secondary-navigation-block">
+      {menuItems.length > 0 ?
+        <div className="secondary-navigation-menu">
+          <ul className="secondary-navigation-item">
```

Comment:
This is really nit-picky but shouldn't the classname here be `secondary-navigation-items` with an 's' since it's the container?